frontend/src/main/java/com/google/tripmeout/frontend/servlet/TripServlet.java

```diff
@@ -0,0 +1,58 @@
+package com.google.tripmeout.frontend.servlet;
+
+import com.google.gson.Gson;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.google.tripmeout.frontend.TripModel;
+import com.google.tripmeout.frontend.error.TripMeOutException;
+import com.google.tripmeout.frontend.error.TripNotFoundException;
+import com.google.tripmeout.frontend.storage.InMemoryTripModelStorage;
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@Singleton
+public final class TripServlet extends HttpServlet {
+  // The Trip get and delete servlet
+  private static final long serialVersionUID = 1L;
+  private final InMemoryTripModelStorage storage;
+  private static final String APPLICATION_JSON_CONTENT_TYPE = "application/json";
+  private final Gson gson;
+
+  @Inject
+  public TripServlet(final InMemoryTripModelStorage storage, Gson gson) {
+    this.storage = storage;
+    this.gson = gson;
+  }
+
+  @Override
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response)
+      throws IOException {
+    try {
+      final String tripId = TripName.fromRequestUri(request.getRequestURI()).id();
+      TripModel theTrip = storage.getTrip(tripId);
+      response.setContentType(APPLICATION_JSON_CONTENT_TYPE + ";");
```

Comment:
Might as well include the `;` in the content type static string since we're just writing it out every time. Alternatively, drop it entirely--it's only technically required when you add [optional parameters](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#Structure_of_a_MIME_type) afterward.

Reply:
yeah, I think he got this from my code where I was also checking the request content type (which doesn't always have the semicolon) with the same string.